### PR TITLE
Update emails

### LINKED ISSUE #1056: Design & Content Revisions to Emails
Following after V2 launch, we need to update the following emails with revised content and updated design. 

**1. Welcome Email with historical report of past breaches. Formerly referred to as "Full Report."**
Users receive this automated email:
- Upon verifying their email address to create a Firefox Account (new FxA users)
- Upon logging in to their Firefox Account from Monitor (existing FxA users)
- Upon verifying a secondary email address in Firefox Monitor. 
- Users may be signing up from the What’s New page. Users may have spent little to no time on the website. 

Considerations: 
- How do we note to users that sensitive breaches are only visible to them?

**2. Breach alert email**
If a breach is added to HIBP that contains this user’s email address, they will receive this automated email. Users may have signed up from the What’s New page, meaning they’ve spent little to no time on the monitor.firefox.com.

Considerations:

- Would a sensitive breach email need to communicate anything different than a a non-sensitive breach? 

**3. Verify your email email**
Update copy and branding.

### PR DESCRIPTION
**Pull request by lesleyjanenorton**
- Updates verification email
- Updates breach report emails. There are now three versions (no found breaches, less than 4 found breaches, and 4+ found breaches).
- Updates the breach alert email. There are now two versions (sensitive, non-sensitive).
- Updates and restyles the /email-l10n routes to use new strings and match the new emails.

Fixes #1056

### CODE REVIEW

File: locales/en/email-strings.ftl [hunk 1]
```diff
@@ -88,3 +88,73 @@ share-by-email-message =
     1. Go to {"https://monitor.firefox.com"} and search your email.
     2. See if your online accounts have been exposed in a data breach.
     3. Get tips from {-product-name} about what to do next.
+
+
+# Unsubscribe link in email.
+email-unsub-link =  Unsubscribe
+
+email-footer-blurb = You’re receiving this email because you signed up for {-product-name} 
+  alerts. No longer want these emails? { $unsubLink }. This is an automated email. For support, visit { $faqLink }.
```
Comment (flodolo) [inline locales/en/email-strings.ftl]: Can you add a comment to explain what's the text used to replace the two variables?

File: locales/en/email-strings.ftl [hunk 2]
```diff
@@ -88,3 +88,73 @@ share-by-email-message =
...
+
+email-verify-footer-copy = You’re receiving this email because you signed up for {-product-name} alerts. 
+  This is an automated email. For support, visit { $faqLink }.
```
Comment (flodolo) [inline locales/en/email-strings.ftl]: Same here: a comment to explain what will replace `$faqLink`

File: locales/en/email-strings.ftl [hunk 3]
```diff
@@ -88,3 +88,73 @@ share-by-email-message =
...
+# Subject line of email
+email-subject-found-breaches = {-product-name} found your info in these breaches
+
+# Subject line of email
+email-subject-no-breaches = {-product-name} found no known breaches
+
+# Subject line of email
+email-subject-verify = Verify your email for {-product-name}
+
+learn-more-about-fxm = Learn more about { $fxmLink }
```
Comment (flodolo) [inline locales/en/email-strings.ftl]: Same here: what replaces `$fxmLink`?

**Comment by pdehaan** [comment]
@lesleyjanenorton Needs a rebase.

File: template-helpers/emails.js [hunk 4]
```diff
@@ -8,27 +8,78 @@ const { makeBreachCards } = require("./breaches");
...
+function emailBreachStats(args) {
+  const locales = args.data.root.supportedLocales;
+  const userBreaches = args.data.root.unsafeBreachesForEmail;
+  let numPasswordsExposed = 0;
+
+  userBreaches.forEach(breach => {
+    if (breach.DataClasses.includes("passwords")) {
+      numPasswordsExposed ++;
+    }
+  });
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply template-helpers/emails.js]: Much prettier!

File: template-helpers/emails.js [hunk 5]
```diff
@@ -8,27 +8,78 @@ const { makeBreachCards } = require("./breaches");
...
+  // garble email address so that email clients won't turn it into a link
+  recipientEmail = recipientEmail.replace(/([@.:])/g, "<span>$1</span>");
+  return `<span class="rec-email text-bold" style=" ${optionalDisplayProperty} font-weight: 700; color: #9059ff; font-family: sans-serif; text-decoration: none;"> ${recipientEmail}</span>`;
+}
+
+
+function getEmailHeader(args) {
+  const locales = args.data.root.supportedLocales;
+  const emailType = args.data.root.whichPartial;
+  const recipientEmail = args.data.root.recipientEmail;
```
[the remark this answers is not in the document]
Comment (lesleyjanenorton) [reply template-helpers/emails.js]: Also much prettier!

**Comment by groovecoder** [comment]
Merged. The string should be available for translation on pontoon, and once they are merged they can best viewed on the heroku app here: https://fx-breach-alerts.herokuapp.com/email-l10n/

**Pull request merged**